handbook/company/leadership.md

```diff
@@ -116,31 +114,28 @@ Consultants aren't required to do any of those things.
-
-> _**Note:** The Docusign template labeled "Contractor agreement (US)" is actually used for both consultants and [core team members in the US who are classified as 1099 contractors or billed corp-to-corp as vendors](#hiring-a-new-team-member).  You may also sometimes hear this referred to as Fleet's "Consulting agreement". Same thing._
-
-To send a US consulting agreement, you'll need the new consultant's name, the term of the service, a summary of the services provided, and the consultant's fee. 
+To send a consulting agreement, you will need to [submit a new consultant onboarding request](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-business-operations&projects=&template=new-consultant-onboarding.md&title=New+US%2Finternational+consultant%3A+_____________) to the business operations team. They will then peform the steps needed to bring aboard a new consultant.
 
-There are some defaults that we use for these agreements:
-   - Term: Default to one month unless otherwise discussed.
-   - Services rendered: Copy and paste from the [language in this doc](https://docs.google.com/document/d/1b5SGgYEHqDmq5QF8p29WWN3it3XJh3xRT3zG0RdXARo/edit)
-   - Work will commence and complete by dates: Start date and end of term date
-   - Fee: Get from the consultant.
-   - Hours: Default to 10 hr/week.
-   - All US consultants track their hours weekly in Gusto.
+You will be asked to provide the following details:
+- Consultant's name (or business name)
+- Consultant's email
+- Term of service (default is 1 month, 10hrs/week)
+- What will they be doing
+- Who will be managing the output (approving hours) for the consultant
+- Consultant's fee (if no rate conversation has happened, BizOps will help negotiate)
+- Any specific tools the consultant will need
 
-Then hit send!  After all of the signatures are there in Docusign, automation will trigger that uploads the completed document to the appropriate Google Drive folder, and that makes a Slack message appear in the `#help-classified` channel.
-
-Finally, create a [custom request](https://fleetdm.com/handbook/business-operations#intake) titled "New US consultant: _____________" and request that this new consultant be registered with Fleet.  (Business Operations will receive this request and take care of next steps, which include things like providing a place for the company to report their hours weekly in the KPIs sheet, and providing access to Slack and any relevant company tools.)
+If the consultant is international, you will also provide:
+- Country of residence
+- Consultant's date of birth
 
 #### Updating a consultant's fee
  - Direct message Mike McNeil with hourly rate change information.
  - After CEO approval, Mike McNeil will issue a new contractor agreement with the updated fee via DocuSign.
 
+<!-- TODO: Update to direct to new BizOps process, and ensure there is clarity between what is "CEO task" and what is "BizOps task"
```

Comment:
Instead of this TODO, make this whole section just say "Send a DM to Joanne Stableford in Slack."